src/app/public/modules/summary-actionbar/summary-actionbar.component.html

```diff
@@ -0,0 +1,60 @@
+<div
+  class="sky-summary-actionbar"
+  [ngClass]="{
+    'sky-summary-actionbar-summary-collapsable': summaryCollapseMode, 'sky-summary-actionbar-modal': inModalFooter
+  }"
+>
+  <div
+    class="sky-summary-actionbar-actions"
+  >
+    <ng-content
+      select="sky-summary-actionbar-actions"
+      ></ng-content>
+  </div>
+  <ng-container *ngIf="summaryContentExists()">
+    <div
+      class="sky-summary-actionbar-summary"
+      [@skyAnimationSlide]="slideDirection"
+      (@skyAnimationSlide.start)="summaryTransitionStart()"
+      (@skyAnimationSlide.done)="summaryTransitionEnd()"
+    >
+      <div
+        class="sky-summary-actionbar-summary-body"
+      >
+        <div class="sky-summary-actionbar-summary-items">
+          <ng-content
+            select="sky-summary-actionbar-summary"
+          ></ng-content>
+        </div>
+        <div *ngIf="summaryCollapseMode && !isSummaryCollapsed"
+          class="sky-summary-actionbar-details-collapse"
+        >
+          <button
+            class="sky-btn sky-btn-secondary"
+            type="button"
+            [attr.title]="'skyux_summary_actionbar_hide_summary' | skyLibResources"
+            (click)="hideSummarySection()"
+          >
+            <sky-icon
+              icon="chevron-down"
+            ></sky-icon>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div *ngIf="isSummaryCollapsed && summaryCollapseMode"
+      class="sky-summary-actionbar-details-expand"
+    >
+      <button
```

Comment:
I believe this button will need aria-controls and aria-expanded set targeting the expandable section and tied to its state.
Everything else looks alright to me a11y-wise. We probably still want to have Matt Gregg Look over the component

Reply:
Done